src/state/models/ui/preferences.ts

```diff
@@ -95,16 +100,35 @@ export class PreferencesModel {
         // default to the device languages
         this.contentLanguages = deviceLocales
       }
-      // check if post languages in preferences exist, otherwise default to device languages
+      if (hasProp(v, 'postLanguage') && typeof v.postLanguage === 'string') {
+        this.postLanguage = v.postLanguage
+      } else {
+        /*
+         * Check if the OLD `postLanguages` in preferences exist, set new field
+         * to old value if exists, otherwise use deviceLocales
+         */
```

Comment:
now that I think about this, it may not be worth looking for the old postLanguages. This is a small enough UI affordance that we could "forget" what was used last and it's not that big of a deal

Reply:
Sgtm!